code/modules/clothing/clothing.dm

```diff
@@ -454,7 +454,7 @@ BLIND     // can't see anything
 	slot_flags = SLOT_FEET
 
 	var/blood_state = BLOOD_STATE_NOT_BLOODY
-	var/list/bloody_shoes = list(BLOOD_STATE_HUMAN = 0, BLOOD_STATE_XENO = 0, BLOOD_STATE_NOT_BLOODY = 0)
+	var/list/bloody_shoes = list(BLOOD_STATE_HUMAN = 0, BLOOD_STATE_XENO = 0, BLOOD_STATE_NOT_BLOODY = 0, BLOOD_STATE_BASE_ALPHA = 150)
```

Comment:
again, shouldn't this default to 255? else it will make all blood translucent?

Reply:
no the variable should default to BLOODY_FOOTPRINT_BASE_ALPHA this is what Im using it to alter